packages/compiler/templates/__snapshots__/library-ts/lib/decorators.tsp

```diff
@@ -0,0 +1,11 @@
+import "../dist/src/decorators.js";
+
+using TypeSpec.Reflection;
+
+namespace LibraryTs;
+
+/**
+ * Provide an alternate name for an operation.
+ * @param name The alternate name.
+ */
+extern dec alternateName(target: Operation, name: valueof string);
```

Comment:
I wonder if there is risk that people think this decorator does something and can't delete it? Could rename to something obviously placeholder-y like "doSomething"? Just a thought.

Reply:
Could also make the comment something like "An example decorator"

Reply:
Yeah I was trying to show a "real" use case but maybe I can prefix with `placeholder_`,  `example_` or `demo_`

Reply:
This is a great addition. There are some conventions we may have taken for granted such as `name` and `$name` in tsp and ts respectively. Also a brief intro about mechnesims various pieces in the projects can really help library writers with a short write-up in the doc section. We can provide a `@dev` section with the link in the comment section. Can we have a separate issue to take that if not in this PR?